t/t4254-am-corrupt.sh

```diff
@@ -60,7 +60,13 @@ test_expect_success setup '
 test_expect_success 'try to apply corrupted patch' '
 	test_when_finished "git am --abort" &&
 	test_must_fail git -c advice.amWorkDir=false am bad-patch.diff 2>actual &&
-	echo "error: git diff header lacks filename information (line 4)" >expected &&
+	echo \
+"error: 
```

Comment:
As you saw, the `check-whitespace` job is really unhappy with that trailing space. There's code at the beginning of the file that provides inspiration how to deal with it:

```diff
diff --git a/t/t4254-am-corrupt.sh b/t/t4254-am-corrupt.sh
index 3de909a5f92..265bcc41bbd 100755
--- a/t/t4254-am-corrupt.sh
+++ b/t/t4254-am-corrupt.sh
@@ -60,8 +60,9 @@ test_expect_success setup '
 test_expect_success 'try to apply corrupted patch' '
 	test_when_finished "git am --abort" &&
 	test_must_fail git -c advice.amWorkDir=false am bad-patch.diff 2>actual &&
+	space=" " &&
 	echo \
-"error: 
+"error:$space
 	could not find header
 caused by:
 	could not find file diff header
```